scripts/confirm_sns_subscription.py

```diff
@@ -0,0 +1,98 @@
+"""
+This script prompts the user for an AWS SNS subscription confirmation link and
+confirms the subscription with AuthenticateOnUnsubscribe enabled. Without
+AuthenticateOnUnsubscribe, anyone with an unsubscribe link would be able to
+delete the subscription, regardless if the subscription was for a group email
+address.
+"""
+import logging
+from typing import (
+    Optional,
+)
+
+from furl import (
+    furl,
+)
+
+from azul import (
+    config,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+sns_client = aws.sns
+
+
+def main() -> None:
+    if config.disable_monitoring:
+        print('SNS subscription confirmation skipped, monitoring is disabled.')
+    elif subscription_pending_confirmation():
+        url = prompt_for_subscription_url()
+        if url is None:
+            print('No URL given, skipping confirmation.')
+        else:
+            confirm_subscription(url)
+    else:
+        print('SNS subscription confirmation skipped, no pending subscriptions found.')
+
+
+def subscription_pending_confirmation() -> bool:
+    """
+    Return True if there is at least one pending subscription
+    """
+    paginator = sns_client.get_paginator('list_subscriptions')
+    for page in paginator.paginate():
+        for subscription in page['Subscriptions']:
+            if subscription['SubscriptionArn'] == 'PendingConfirmation':
+                return True
+    return False
+
+
+def prompt_for_subscription_url() -> Optional[str]:
+    """
+    Prompt user for a confirmation URL
+    """
+    print()
+    print('This process will confirm a pending SNS subscription and configure\n'
+          'it so that an unsubscribe action will require AWS authentication.\n'
+          'Please open the "Subscription Confirmation" email, copy the\n'
+          '"Confirm subscription" link address, and paste the value here.\n'
+          'If you do not have a pending confirmation, this step can be skipped.')
```

Comment:
```suggestion
    print('To confirm the pending subscription of <email> to the SNS topic that <describe the topic>, please open the "Subscription Confirmation" email, copy the\n'
          '"Confirm subscription" link address, and paste the value here.\n'
          'If you do not have a pending confirmation, this step can be skipped.')
```

And why would they not have the email. What does it mean "to skip this step"? What should they enter?